main.tf

```diff
@@ -245,6 +245,32 @@ module "cache" {
   cache_expiration_days                = var.cache_expiration_days
 }
 
+resource "aws_s3_bucket_policy" "cache" {
```

Comment:
This policy is already created via the cache module, please check `modules/cache/policies/cache.json`